README.md

```diff
@@ -49,7 +49,7 @@ To learn React, check out the [React documentation](https://reactjs.org/).
 
```

Comment:
- [ ] Add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Reply:
This change has been done well